pep-0012.rst

```diff
@@ -148,6 +149,25 @@ directions below.
 
 - Send your PEP submission to the PEP editors at peps@python.org.
 
+For reference, here are all of the possible header fields::
```

Comment:
Most of these have a specific format. Maybe (in a follow-up PR?) we should describe those? Or is there reference documentation for pep2html.py already?

Reply:
We can put examples in there.